appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/ejb-jar_4_1.xsd

```diff
@@ -0,0 +1,3360 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="https://jakarta.ee/xml/ns/jakartaee"
+            xmlns:jakartaee="https://jakarta.ee/xml/ns/jakartaee"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            elementFormDefault="qualified"
+            attributeFormDefault="unqualified"
+            version="4.1">
```

Comment:
I don't understand. EJB 4.1 is only planned for Jakarta EE 12 (2026). Is this some Payara extension?

> This PR is to add samples to test the new xml tag managed-executor from files configuration

The description is about samples, but it adds this file to final distributions as well.

The line 32 below says:
> This is the XML Schema for the Enterprise Beans 4.0 deployment descriptor.

and it includes jakartaee_10.xsd. But there was no separate EJB release in Jakarta EE 10. :confused: 

Weird.

Maybe FISH-6037 has some references? May I ask for them to be shared, please?

Reply:
Huh, yeah, this is odd.

FISH-6037 is adding support for `@ManagedExecutorDefinition`, and this seemed to slurp up some additional work of adding support for defining it in various deployment descriptors (web.xml, application.xml, and ejb-jar.xml). I'd have to go look at the specs, but I suspect these are Payara-specific enhancements